crates/sqlexec/src/planner/session_planner.rs

```diff
@@ -887,7 +887,8 @@ impl<'a> SessionPlanner<'a> {
                 let has_header = storage_options
                     .inner
                     .get("has_header")
-                    .map(|val| val.parse::<bool>().unwrap_or_else(|| false))?;
+                    .map(|val| val.parse::<bool>().unwrap_or(false))
```

Comment:
I just opened a different PR for the table function to have this default to true

Reply:
Sure, will change it here as well.